b2share/modules/schemas/root_schemas/root_schema_v0.json

```diff
@@ -5,137 +5,209 @@
+                            "title": "Type",
+                            "enum": ["Abstract", "Methods", "SeriesInformation", "TableOfContents", "TechnicalInfo", "Other"]
+                        }
+                    },
+                    "required": ["description", "description_type"]
+                },
+                "uniqueItems": true
             },
             "version": {
                 "title": "Version",
                 "description": "Denote the version of the resource.",
                 "type": "string"
             },
-            "publisher": {
-                "title": "Publisher",
-                "description": "The entity responsible for making the resource available, either a person, an organization, or a service.",
+            "contact_email": {
+                "title": "Contact Email",
+                "description": "Contact email information for this record.",
                 "type": "string",
-                "default": "https://b2share.eudat.eu"
+                "format": "email"
             },
-            "language": {
-                "title": "Language",
-                "description": "The name of the language the document is written in.",
+
+            "open_access": {
+                "title": "Open Access",
+                "description": "Indicate whether the record's files are publicly accessible or not. In case of restricted access the uploaded files will only be accessible by the record's owner and the community administrators. Please note that the record's metadata is always publicly accessible. ",
+                "type": "boolean"
+            },
+            "embargo_date": {
+                "title": "Embargo Date",
+                "description": "The date marking the end of the embargo period. The record will be marked as open access on the specified date at midnight. Please note that the record metadata is always publicly accessible, and only the data files can have private access.",
                 "type": "string",
-                "default": "en"
+                "format": "date-time"
+            },
+
+            "licence": {
+                "title": "Licence",
+                "description": "Specify the license under which this data set is available to the users (e.g. GPL, Apache v2 or Commercial). Please use the License Selector for help and additional information.",
+                "type": "string"
+            },
+
+            "community": {
+                "title": "Community",
+                "description": "The community to which the record has been submitted.",
+                "type": "string"
             },
 
             "community_specific": {
                 "type": "object"
             },
+
+
+            "publication_state": {
+                "title": "Publication State",
+                "description": "State of the publication workflow.",
+                "type": "string",
+                "enum": [ "draft", "submitted", "published", "deleted" ]
```

Comment:
There is no need for a "deleted" marker in `publication_state`. [invenio-records already has a way to know when a record has been deleted](https://github.com/inveniosoftware/invenio-records/blob/81711bfb556ffa92c395b8298479046ff0ed6d68/invenio_records/models.py#L65). And the way to delete a record is not to do a PATCH of `publication_state` but to directly use the HTTP DELETE command.